### PR TITLE
Update apply.md

### PR DESCRIPTION
**Pull request by sofer**
@iteles as discussed this morning, more encouragement for applicants.

### CODE REVIEW

File: apply.md [hunk 1]
```diff
@@ -39,7 +39,9 @@ We aim to accept around four applicants after each deadline. We do it this way,
 
 You can contribute to the community by communicating with other members of the community: posting questions, answering questions, posting your Codewars kata, or any other way that you like.
 
-If you want help making progress with the prerequisites and you live in London, you might want to consider signing up for our [Monday evening meetups](http://www.meetup.com/founderscoders/).
+If you have no coding experience, getting to 6 kyu (or higher) in Code wars will take time. If you are an absolute beginner, you might need to study for 200 hours or more to meet the required standard.
```
Comment (iteles) [inline apply.md]: I think 200 hours is really pretty scary. Could we say 'a considerable amount of time'?

**Comment by iteles** [comment]
Will this be on the website too? Not many people will see this on our github when they're thinking of applying :)

**Comment by harrygfox** [comment]
@sofer @iteles Sounds scary to me - it's compiled to html and is [here](http://foundersandcoders.org/apply.html) on the website

**Comment by iteles** [comment]
@harrygfox @sofer D'oh! I'm far too used to reviewing PRs on the playbook, didn't even notice this was the fac github.io!!

**Comment by sofer** [comment]
@harrygfox @iteles thank you so much for your feedback. I have amended the copy.

**Pull request merged**